sdk/src/main/java/io/opentelemetry/sdk/metrics/DoubleSumAggregator.java

```diff
@@ -18,21 +18,26 @@
 
 import com.google.common.util.concurrent.AtomicDouble;
 
-final class DoubleSumAggregator implements Aggregator.DoubleAggregator<DoubleSumAggregator> {
+final class DoubleSumAggregator implements Aggregator<DoubleSumAggregator> {
   // TODO: Change to use DoubleAdder when changed to java8.
-  private final AtomicDouble value;
+  private final AtomicDouble current;
 
   DoubleSumAggregator() {
-    this.value = new AtomicDouble();
+    this.current = new AtomicDouble(0.0);
   }
 
   @Override
   public void merge(DoubleSumAggregator other) {
-    this.value.addAndGet(other.value.get());
+    current.getAndAdd(other.current.get());
```

Comment:
I would prefer to have `this.current` for readability.
```suggestion
    this.current.getAndAdd(other.current.get());
```

Reply:
done.